core/codegen/src/attribute/route/parse.rs

```diff
@@ -209,9 +211,11 @@ impl Route {
             })
             .collect();
 
+        let docstring = String::from_attrs("doc", &handler.attrs)?.join("\n");
```

Comment:
I believe we need it, because Rust comments are markdown, which is also supported for OpenApi and RAML. Markdown is a newline sensitive format if you will: `# title \n remainder\n` is not the same as `# title remainder`.

Reply:
My question is really where rustdoc already inserts a new line or whether it infers one based on having multiple attributes. You should be able to easily check. ￼￼￼We only need the new line here if it is inferred.￼